ModernWpf/Controls/Primitives/PointerUpDownHelper.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Input;
+using System.Windows.Media.Animation;
+
+namespace ModernWpf.Controls.Primitives
+{
+    public static class PointerUpDownHelper
```

Comment:
Could this class be made internal?

Reply:
Do you also want the **Planerator** and **PlaneratorHelper** to be `internal` classes? Because they are unnecessary to this library since you are trying to replicate **WinUI**

Reply:
If you elaborate your request, I will make those changes within a day